R/data.R

```diff
@@ -69,17 +69,17 @@
 #'  \item{percasian}{Percent Asian.}
 #'  \item{percother}{Percent other races.}
 #'  \item{popadults}{Number of adults.}
-#'  \item{perchsd}{}
+#'  \item{perchsd}{Percent with high school diploma.}
 #'  \item{percollege}{Percent college educated.}
-#'  \item{percprof}{Percent profession.}
-#'  \item{poppovertyknown}{}
-#'  \item{percpovertyknown}{}
-#'  \item{percbelowpoverty}{}
-#'  \item{percchildbelowpovert}{}
-#'  \item{percadultpoverty}{}
-#'  \item{percelderlypoverty}{}
-#'  \item{inmetro}{In a metro area.}
-#'  \item{category}{}
+#'  \item{percprof}{Percent with professional degree.}
+#'  \item{poppovertyknown}{Population with known poverty status.}
+#'  \item{percpovertyknown}{Percent of population with known poverty status.}
+#'  \item{percbelowpoverty}{Percent of people below poverty line.}
+#'  \item{percchildbelowpovert}{Percent of children below poverty line.}
+#'  \item{percadultpoverty}{Percent of adults below poverty line.}
+#'  \item{percelderlypoverty}{Percent of elderly below poverty line.}
+#'  \item{inmetro}{County considered in a metro area.}
+#'  \item{category}{Miscellaneous.}
```

Comment:
I could not find out what this was but I didn't want to leave this blank, so I just gave it this name. Happy to remove this if it is pointless.